app/models/responsible_body.rb

```diff
@@ -32,4 +32,27 @@ def who_will_order_devices_label
       humanized_type.capitalize
     end
   end
+
+  def self.find_by_computacenter_urn!(cc_urn)
```

Comment:
I think we should pull this and the subsequent methods into a module and include it into this class, to group these and make it explicit that these are standalone and are related to CC.

Reply:
Yeah, I originally tried having these in a standalone class in the `Computacenter::` namespace, but it didn't quite work - a module may work better, will give it a try

Reply:
Done, that makes it cleaner 👍